be/src/vec/common/schema_util.cpp

```diff
@@ -174,6 +171,15 @@ Status cast_column(const ColumnWithTypeAndName& arg, const DataTypePtr& type, Co
         return Status::OK();
     }
 
+    auto function = SimpleFunctionFactory::instance().get_function("CAST", arguments, type);
```

Comment:
why change the code order ?

Reply:
since some branch does not need to execute function，so get the function is useless